examples/nlp/text_classification_tf_keras/adaptive.yaml

```diff
@@ -0,0 +1,21 @@
+description: text_classification_tf_adaptive
+hyperparameters:
+  global_batch_size: 32
+  dense1: 5
+  embedding_dim:
+    type: int
+    minval: 16
+    maxval: 32
+records_per_epoch: 60000
+searcher:
+  name: adaptive_asha
+  metric: val_sparse_categorical_accuracy
+  smaller_is_better: false
+  max_length:
+    epochs: 5
+  max_trials: 10
+environment:
+  image:
+    gpu: determinedai/environments:cuda-11.0-pytorch-1.7-lightning-1.2-tf-2.4-gpu-0.12.0
```

Comment:
I'd suggest using one single image versioning scheme, i.e. either semantic (0.12.0) or hashed (baefbf7), but not mix them. 
There's probably no specific reason to use this particular hash, so probably using latest `-0.13.0` for both gpu and cpu images should work the best.

Reply:
updated, thanks!